packages/core-forger/src/client.ts

```diff
@@ -37,93 +44,48 @@ export class Client {
 
     /**
      * Send the given block to the relay.
-     * @param  {(Block|Object)} block
-     * @return {Object}
      */
-    public async broadcast(block) {
+    public async broadcast(block: models.IBlockData): Promise<any> {
         this.logger.debug(
             `Broadcasting forged block id:${block.id} at height:${block.height.toLocaleString()} with ${
                 block.numberOfTransactions
             } transactions to ${this.host}`,
         );
 
-        return this.__post(`${this.host}/internal/blocks`, { block });
+        return this.post(`${this.host}/internal/blocks`, { block });
     }
 
     /**
      * Sends the WAKEUP signal to the to relay hosts to check if synced and sync if necesarry
      */
-    public async syncCheck() {
-        await this.__chooseHost();
-
+    public async syncCheck(): Promise<void> {
         this.logger.debug(`Sending wake-up check to relay node ${this.host}`);
-
-        try {
-            await this.__get(`${this.host}/internal/blockchain/sync`);
-        } catch (error) {
-            this.logger.error(`Could not sync check: ${error.message}`);
-        }
+        await this.get(`${this.host}/internal/blockchain/sync`);
     }
 
     /**
      * Get the current round.
-     * @return {Object}
      */
-    public async getRound() {
-        try {
-            await this.__chooseHost();
-
-            const response = await this.__get(`${this.host}/internal/rounds/current`);
-
-            return response.body.data;
-        } catch (e) {
-            return {};
-        }
+    public async getRound(): Promise<any> {
```

Comment:
Should add a struct/interface for the expected data we want to return instead of using `any`.